plugins/GSdx/GLLoader.cpp

```diff
@@ -276,9 +276,9 @@ namespace GLLoader {
 			mesa_amd_buggy_driver = intel_buggy_driver = true;
 
 #ifdef _WIN32
-		buggy_sso_dual_src = intel_buggy_driver || fglrx_buggy_driver || legacy_fglrx_buggy_driver;
+		buggy_sso_dual_src = intel_buggy_driver || legacy_fglrx_buggy_driver;
```

Comment:
Couldn't it be setup to works sort of like compiler version defines? So checks would more like `if (amd_driver_ver > x || amd_driver_ver < y) applyHack();` ?
It'd probably be a bit harder to maintain, but shouldn't be that bad, since driver updates are not that frequent and versioning scheme changes are even less so.